pkgs/development/haskell-modules/configuration-arm.nix

```diff
@@ -36,6 +36,7 @@ self: super: {
 
   # Similar to https://ghc.haskell.org/trac/ghc/ticket/13062
   happy = dontCheck super.happy;
+  happy_1_19_12 = dontCheck super.happy_1_19_12;
```

Comment:
can you wrap this in `doDistribute`? Then we'll get binary cache for this as well.